### DISCUSSION TITLE
What is R5 (100KR) for

### DISCUSSION THREAD

**Discussion by Shoey5** [frntc/SIDKick-pico#64] [Q&A]
I used the BOM on PCBWay while ordering the PCB for SKpico with PWM or external DAC to order parts. I just built it but I noticed that the BOM here lists R5 (100KR). Looks like it says optional but just wondering what it's for and why the discrepancy between the two BOMs and is it actually optional or required?

**Comment by frntc** [comment]
R5 is a pull-up resistor for the RP2040's RUN (actually a reset-) pin. It seems, but I wasn't sure if such a resistor is present on all Pico(-clone) boards, so I added R5 in case it's needed. If you don't experience any problems (as I did) you can skip it.
